Index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>Figma-portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" id="logo">My Logo</a>
+      <nav>
+        <ul id="nav">
+          <li class="nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-link"><a href="#">About</a></li>
+          <li class="nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="mobile-nav">
+          <div id="menu">
+            <i class="bar"></i>
+            <i class="bar"></i>
+            <i class="bar"></i>
+          </div>
+          <ul id="link">
+            <li class="mobile-link">
+              <a href="#">Portfolio</a>
+            </li>
+            <li class="mobile-link">
+              <a href="#">About</a>
+            </li>
+            <li class="mobile-link">
+              <a href="#">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
+      <div id="head-container">
+        <h1 class="head">
+          I'm John.   </br>
+          Glad to see </br>
+          you!
+        </h1>
+        <p class="resume">
+          I’m a software developer! I can help you </br>
+          build a product, feature orwebsite Look </br>
+          through some of my work and </br>
+          experience. If you like what you see and </br>
+          have a project you need coded, don’t </br>
+          hesitate to contact me!
+        </p>
+        <h2 class="connect">LET’S CONNECT</h2>
+        <ul class="social-media-icons">
+          <li>
+            <a rel="noopener" href="https://twitter.com/jovasal81" target="_blank">
+              <img src="./images/twitter.png" alt="twitter" />
+            </a>
+          </li>
+          <li>
+            <a rel="noopener" href="https://www.linkedin.com/in/john-alonso-vasquez-salcedo-95749632/" target="_blank">
+              <img src="images/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a rel="noopener" href="https://medium.com/@jovasal" target="_blank">
+              <img src="images/medium.png" alt="medium" />
+            </a>
+          </li>
+          <li>
+            <a rel="noopener" href="https://github.com/jovasal" target="_blank">
+              <img src="images/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a rel="noopener" href="https://wellfound.com/john-vasquez-12" target="_blank">
+              <img src="images/angel.png" alt="angel" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <main class="secpa">
```

Comment:
- [ ] As requested by the previous CR, please 🙏🏽 correct the margins to center the content of the works section to be similar to the Figma design 👇🏽 

https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/bede7af1-9871-4ef7-b3ac-a510f9f4b72d

Reply:
Hi @franclobo I understand. 
Margin we fixed and actually are running well:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/081bb59f-b4ab-4070-88b0-b4144db7aef0)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/85e156c2-c61a-490e-af0b-c2b304326259)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/d8ed9095-d062-415a-9b60-7dfe89c86aef)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/2a2c1501-d884-4346-85b4-56617a1c95d5)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/3d2a2c89-a0b7-4b23-871d-3af17a972a75)